Tests/ArcGISToolkitTests/ARTests.swift

```diff
@@ -0,0 +1,188 @@
+// Copyright 2023 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import SwiftUI
+import XCTest
+@testable import ArcGISToolkit
+
+@MainActor final class ARTests: XCTestCase {
+    func testFlyoverLocationInitWithDefaults() throws {
+        let sceneView = SceneView(scene: Scene())
+        
+        let view = FlyoverSceneView(
+            initialLocation: .init(
+                latitude: 0,
+                longitude: 0
```

Comment:
it would probably be better to choose numbers other than 0 because 0 is sometimes a default and that could potentially imply "not-set"